### DISCUSSION TITLE
Add more descriptive image to docs - especially Design section.

### DISCUSSION THREAD

**Discussion by 0xdk89** [MystenLabs/walrus-docs#106] [General]
We need to add more descriptive images to the Walrus docs, especially in the Design -> Architecture section.
To show the connection between Blob, Slivers, Shards, Storage Nodes.

**Comment by 0xdk89** [comment]
Something just like this one
![Screenshot 2024-08-15 at 15 37 32](https://github.com/user-attachments/assets/c94ad45c-4ee1-4116-a384-652cb8b0d961)

**Comment by mlegner** [comment]
Thank you for the suggestion. I've been wanting to do this for a while but never found the time. I now created a separate issue for this: #109.

**Reply by 0xdk89** [reply]
Ok, let me contribute some pics to this. I'm researching Walrus, so I also need to verify my knowledge. Thanks.
